scipy/sparse/construct.py

```diff
@@ -791,5 +791,20 @@ def rand(m, n, density=0.01, format="coo", dtype=None, random_state=None):
     -----
     Only float types are supported for now.
 
+    See Also
+    --------
+    scipy.sparse.random : Equivalent function when passing `data_rvs=None`
```

Comment:
Is this the correct way to clarify this? (And is there a reason for `rand` to exist int the first place?)